apps/ledger-live-desktop/tests/specs/speculos/manage.ledgersync.spec.ts

```diff
@@ -0,0 +1,43 @@
+import { test } from "../../fixtures/common";
+import { addTmsLink } from "tests/utils/allureUtils";
+import { getDescription } from "../../utils/customJsonReporter";
+import { AppInfos } from "tests/enum/AppInfos";
+
+// Use a single value instead of an array
+const app: AppInfos = AppInfos.LS;
+
+test.describe(`[${app.name}] Sync Accounts`, () => {
+  test.use({
+    userdata: "ledgerSync",
+    speculosApp: app,
+  });
+
+  test("Synchronize one instance then delete the backup", async ({ app, page }) => {
+    const description = getDescription(test.info().annotations).split(", ");
+    await addTmsLink(description);
+
+    // Step 1: Validate the presence of the sync entry point
+    await app.settings.goToSettings();
+    await app.settings.goToGeneralSettings();
+    await app.settings.openManageLedgerSync();
+    await app.ledgerSync.expectSyncAccountsButtonExist();
+
+    // Step 2: Perform synchronization and validate the process
+    await app.ledgerSync.SyncAccounts();
+    await app.speculos.confirmOperationOnDevice("Log in to");
+    await app.speculos.confirmOperationOnDevice("Enable");
+    await app.ledgerSync.expectSynchronizationSuccess();
+    await app.ledgerSync.closeLedgerSync();
+
+    // Step 3: Validate the number of synchronized instances
+    await app.settings.openManageLedgerSync();
+    await app.ledgerSync.expectNbSyncedInstances(1);
+    await app.ledgerSync.closeLSManagerDrawer();
+
+    // Step 4: Delete backup
+    await app.settings.openManageLedgerSync();
```

Comment:
I don't get why you click on Manage Ledger Sync button two times: once on line 33 and second time here, maybe it's better to click only once and do all those checks?